### ISSUE TITLE
Group authors

### ISSUE THREAD

**Issue by FAtherden-eLife** [substance/texture#1278]
# Description

A group author is an alternative to a singular author, representing a group that has contributed to an article. They are composed of multiple individual contributors, who may or may not be acknowledged by name and whose contribution is counted collectively rather than individually.

### Author
- As an author, I want to be able to edit the name of the group author so that I can correct any errors.
- As an author, I want to be able to see the membership of a group author so that I can check this information is correct.
- As an author, I want to be able to edit the details (names and affiliations) for the members of a group author so that I can correct any mistakes.

### Production staff
- As production staff, I want to be able to add a new group author to the author list so that I can include any missing group authors.
- As production staff, I want to be able to remove a group author from the author list so that I can correct any errors.
- As production staff, I want to be able to add members to the group author so that I can include any missing members.
- As production staff, I want to be able to remove members from the group author so that I can correct any mistakes.

### But what if . . . ?
- 

### Considerations
- It is important to semantically include individual author names when supplied as these are sent to PubMed as contributors (but not direct authors) and the individual author names appear in PubMed searches. This is important to authors for recognition.
- For contributions, competing interests, affiliations and corresponding author status, group authors act identically to normal authors. However unlike normal authors, they do not HAVE to have contributions, competing interests or affiliations.
- For a member of a group author, the following details must be captured:
      - Name
- For a member of a group author, the following details can be captured but are not required:
      - Affiliation
      - Author ID (such as ORCiD)
      - Email address
- On the Reproducibility Project, the group author that served as corresponding author was permitted two email addresses - do we want to keep allowing this? (linked to #20)

## XML requirements
Captured as a `contrib`. It must contain a `collab` element (which contains the plain text name of the group author), and may contain a child `contrib-group` element which contains the members of the group. Affiliations are tagged as a child of the member.
```xml
<contrib corresp="yes">
         <email>....</email>
         <email>...</email>
         <collab>Production team</collab>
         <contrib-group>
                  <contrib>
                           <name>
                                    <surname>Atherden</surname>
                                    <given-names>Frederick P</given-names>
                           </name>
                           <xref ref-type="aff" rid="aff11">11</xref>
                  </contrib>
                  <contrib>
                           <name>
                                    <surname>Gilbert</surname>
                                    <given-names>James F</given-names>
                           </name>
                           <xref ref-type="aff" rid="aff12">12</xref>
                  </contrib>
                  ….
                  <aff id="aff11">
                           <label>11</label>
                           <institution content-type="dept">Department of Technology</institution>
                           <institution>eLife</institution>
                           <addr-line>
                                  <city>Cambridge</city>
                           </addr-line>
                           <country>United Kingdom</country>
                  </aff>   
                  <aff id="aff12">
                           <label>12</label>
                           <institution content-type="dept">Department of Production</institution>
                           <institution>eLife</institution>
                           <addr-line>
                                  <city>Cambridge</city>
                           </addr-line>
                           <country>United Kingdom</country>
                  </aff>         
         </contrib-group>
</contrib>
```

Group author affiliations should be captured in the child `contrib-group`, as these are not at the same level as standard author affiliations.

# Mock ups

This is not required, but if you have mock ups of what you would like to see please provide them here. 

# Proposal

This will be added by the Texture team after the feature request is discussed and agreed.

**Comment by michael** [comment]
@Melissa37 @FAtherden-eLife In which cases do we have affiliations assigned to group members? Do you have a sample of an existing article that displays an affiliation for a group member? Could this be too detailed?

**Comment by FAtherden-eLife** [comment]
@michael Yep, we have articles where group members are assigned affiliations. Examples:
- https://elifesciences.org/articles/43511
- https://elifesciences.org/articles/39944
- https://elifesciences.org/articles/39856
- https://elifesciences.org/articles/37754
